definitions/tools/interval_list_expand.cwl

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "expand interval list regions by a given number of basepairs"
+
+baseCommand: ["/usr/bin/java", "-Xmx16g", "-jar", "/usr/picard/picard.jar", "IntervalListTools"]
+
+arguments:
+    ["OUTPUT=", { valueFrom: $(runtime.outdir)/expanded.interval_list }, "UNIQUE=TRUE"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 16000
```

Comment:
It's good to leave some overhead here between the java `-Xmx` and the memory request.  If Picard really did try to use the full Java quota, it would exceed this amount.  (16 GiB > 16000 MiB!)